docs/book/zones.rst

```diff
@@ -0,0 +1,77 @@
+.. index::
+   single: Zones
+
+Zones
+=====
+
+**Zones** are a part of the :doc:`Addressing </book/addresses>` concept.
+
+Zones and ZoneMembers
+---------------------
+
+**Zones** consist of **ZoneMembers**.
+Three different types of zones are available:
+
+* `country` zone, which consists of countries.
+* `province` zone, which is constructed from provinces.
+* `zone`, which is a group of other zones.
```

Comment:
Should we mention, that it can be any arbitrary chosen zone? For example ZIP code ranges in USA?